packages/media-sound/audacious/audacious.exlib

```diff
@@ -19,6 +19,7 @@ SLOT="0"
 MYOPTIONS="
     dbus [[ description = [ Build mpris backend (can be later used by mpris2 plugin). ] ]]
     gtk [[ description = [ Build the GTK+ 2 GUI ] ]]
```

Comment:
Should probably be moved to gtk3 which is supported now and gtk2 considered end-of-life: https://github.com/audacious-media-player/audacious/commit/09ce8a4181ea18516d16e51fe70a8b9e894ea8a2